README.md

```diff
@@ -1,7 +1,49 @@
 # annotation-service
 Annotation integration service provides geolocation for IPv4 and IPv6 MaxMind databases from Google Cloud Storage.
 
-Annotated geolocation data includes:
+If an annotation request is dated prior to August 2017, location data will be derived from
+MaxMind GeoLiteLatest databases. Otherwise, data will be provided by
+MaxMind GeoLite2 databases. The discrepencies between provided databases  are
+provided below.
+
+MaxMind GeoLiteLatest databases include:
+  1. IP Block File
+    - StartIPNum IPv4
+    - EndIPNum  IPv4
+    - GeonameID 
+  2. Location File
+    - GeonameID
+    - Country Code
+    - City
+    - Postal Code
+    - Logitude
+    - Latitude
+    - MetroCode
+
+MaxMind GeoLite2 databases include:
+  1. IP Block File
+    - IP network (CIDR Format)
+    - GeonameID (identifies end user location)
+    - Registered Country Geoname ID (identifies country where IP address is
+      registered to an ISP)
+    - Latitude
+    - Longitude
+  2. Location File
+    - GeonameID
+    - Continent Code
+    - Country ISO
+    - Country Name
+    - City Name
+    - Metro Code
+
```

Comment:
geoname_id is the same with registered_country_geoname_id most of time, but with some exceptions. 
Either geoname_id or registered_country_geoname_id could be not available for some IP addresses.